using_images/sti_images/python.adoc

```diff
@@ -10,10 +10,13 @@
 toc::[]
 
 == Overview
-The OpenShift https://github.com/openshift/sti-python/tree/master/3.3[Python]
-image repository contains the sources and Dockerfiles for building various
-versions of Python platform images. The resulting images can be run either by
-Docker or using https://github.com/openshift/source-to-image[STI].
+OpenShift provides https://github.com/openshift/source-to-image[STI]-enabled
+Python images for building and running Python applications.  These images can be
+used with the
+link:../../architecture/core_objects/builds.html#sti-build[STI build strategy];
+the https://github.com/openshift/sti-python[STI builder image] combines with
```

Comment:
Although technically correct, I think we could write this as:
STI builder image assembles your application with any required dependencies to create a new image....
I think combining isn't that clear, that's why I'm proposing assemble, because I don't like repeating like in builder builds, although that's what I'd prefer to show, that the builder is responsible for building, because the process isn't that simple.